src/playlist-maker.js

```diff
@@ -24,13 +34,20 @@ var playlistMaker = function(player, plist) {
     if (plist && isArray(plist)) {
       list = plist.slice();
       player.playlist.currentItem(0);
+
+      window.setTimeout(function() {
```

Comment:
That sounds reasonable. Definitely worth adding to the description in the README.